www/js/app.js

```diff
@@ -773,7 +776,7 @@ define(['jquery', 'zimArchiveLoader', 'util', 'uiUtil', 'cookies','abstractFiles
                     iframeArticleContent.onload = function () {};
                     $("#searchingArticles").hide();
                 };
-                iframeArticleContent.src = dirEntry.namespace + "/" + encodeURIComponent(dirEntry.url);
+                iframeArticleContent.src = '../' + dirEntry.namespace + "/" + encodedUrl;
```

Comment:
I suppose the `../` has been added to remove the `www/` part of URLs? It seems to be a good idea

Reply:
From memory, I think this was necessary to match the scope of the SW, so that the URL is in the same scope whether it is loaded by app.js (as the result of a search), or by the Service Worker. But it was some time ago. I tested it in different scenarios, and this is what worked.